File: core/client/game/src/commonGame/tool/GameDataRegister.cs
using ShineEngine;

/// <summary>
/// 
/// </summary>
[Hotfix]
public class GameDataRegister:DataRegister
{
	public override void regist()
	{
		base.regist();

		add(new BaseDataMaker());

		add(new PlayerPartDataMaker());
		add(new PlayerListDataMaker());
	}
}

File: core/clientLaya/game/src/commonGame/tool/GameDataRegister.ts
namespace Shine
{
    /** 数据注册类 */
    export class GameDataRegister extends DataRegister
    {
        constructor()
        {
            super();
        }

        public regist():void
        {
            super.regist();

            this.add(new BaseDataMaker());

            this.add(new PlayerPartDataMaker());
            this.add(new PlayerListDataMaker());
        }
    }
}